pkgs/development/python-modules/openbabel-bindings/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib, openbabel, python, buildPythonPackage }:
+
+buildPythonPackage rec {
+    pname = "openbabel";
+    version = "3.1.1";
+
+    src = "${openbabel}/lib/python${python.sourceVersion.major}.${python.sourceVersion.minor}/site-packages";
+
+    nativeBuildInputs = [ openbabel ];
+
+    # these env variables are used by the bindings to find libraries
+    # they need to be included explicitly in your nix-shell for
+    # some functionality to work (inparticular, pybel).
+    # see https://openbabel.org/docs/dev/Installation/install.html
+    BABEL_LIBDIR = "${openbabel}/lib/openbabel/3.1.0";
+    LD_LIBRARY_PATH = "$LD_LIBRARY_PATH:${openbabel}/lib";
+
+    meta = with lib; {
+      homepage = "http://openbabel.org/wiki/Main_Page";
+      description = "Python bindings for openbabel";
+      license = licenses.gpl2Plus;
+      maintainers = with maintainers; [ danielbarter ];
+    };
```

Comment:
I think your editor is configured incorrect or does not pickup the editor config. Intonations need to be 2 spaces.